### PR TITLE
ADE20k DeepLabv3 optimized benchmark yaml

### PR DESCRIPTION
**Pull request by Landanjs**
yaml file for optimized ade20k-deeplabv3 benchmark and renames `ade20k_deeplabv3.yaml` -> `ade20k_deeplabv3_unoptimized.yaml`

Primary changes from unoptimized:
- Decoupled WD: slight mIoU degradation, but makes tuning easier
- Batch Size 16 -> 32: mIoU improvement and 1.3x-1.4x reduction in training time
- Weight decay 5e-5 -> 2e-5: mIoU improvement
- Polynomial LR schedule -> Cosine decay: minor mIoU improvement

[Unoptimized mIoU](https://wandb.ai/mosaic-ml/landan-deeplabv3-optimized/groups/Earliest_unoptimized/workspace?workspace=user-landanjs): 44.8 +/- 0.37 (4 seeds)
[Optimized mIoU](https://wandb.ai/mosaic-ml/landan-deeplabv3-optimized/groups/dwd_bs32_lr0.01_wd2e-5_cos_dev/workspace?workspace=user-landanjs): 45.7 +/- 0.095 (4 seeds)

I should write a report/blog post on what worked and what didn't.

### CODE REVIEW

**Review by coryMosaicML — COMMENTED** [review]
LGTM!

**Review by coryMosaicML — APPROVED** [review]
